tests/schema.py

```diff
@@ -271,12 +271,13 @@ class DecimalPrimaryKey(dj.Lookup):
     """
     contents = zip((0.1, 0.25, 3.99))
 
+
 @schema
 class IndexRich(dj.Manual):
     definition = """
     -> Experiment
     ---
-    (first) ->[unique, nullable] User
+    -> [unique, nullable] User.proj(first="username")
```

Comment:
Actually, given that we are keeping forward compatibility, we need to ensure that old syntax still works. Please add back the old syntax to the test.

Reply:
This kept both syntaxes.  The first dependency uses the old syntax.

Reply:
I did leave some dependencies in the old syntax.  `IndexRich` is a new test, so I used the new syntax.  Old tests use old syntax.